mobile/polydodo-mobile.code-workspace

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Remove this file

Reply:
Si on veut ajouter de la config dans `mobile`, faut simplement utiliser le folder `mobile/.vscode`